snapshot/lib/snapshot/runner.rb

```diff
@@ -1,3 +1,4 @@
+# coding: utf-8
```

Comment:
I believe this is the default for Ruby >= 2.0, so I think it is not needed here. What was the intent?